src/main/java/service/BookService.java

```diff
@@ -15,13 +17,22 @@ public void createBook(String bookname, String author, int year,int isbn) {
         bookRepo.addBook(book);
     }
 
-    public boolean findBook(String name,int isbn) {
-        Book book = bookRepo.getBook(name,isbn);
-        return book != null;
+    public ArrayList<Book> findBooks(String name, String author) {
+        Book book = bookRepo.getBook(name, author);
+        if (book != null){
+            return bookRepo.findBooks(book);
+        }
+        return null;
+    }
+
+
+
+    public Book getBook(String name, String author) {
+        return (bookRepo.getBook(name,author));
     }
 
-    public Book getBook(String name,int isbn) {
+    public Book getBookReservation(String name,String author,int isbn) {
```

Comment:
Same as my comment above, I see two methods doing the same thing, one with name and author and the otherone with name, author and ISBN